proposals/nnnn-custom-actor-executors.md

```diff
@@ -35,7 +35,7 @@
 
 As Swift Concurrency continues to mature it is becoming increasingly important to offer adopters tighter control over where exactly asynchronous work is actually executed.
 
-This proposal introduces a basic mechanism for customizing actor  executors. By providing an instance of an executor, actors can influence "where" they will be executing any task they are running, while upholding the mutual excusion and actor isolation guaranteed by the actor model.
+This proposal introduces a basic mechanism for customizing actor  executors. By providing an instance of an executor, actors can influence "where" they will be executing any task they are running, while upholding the mutual execusion and actor isolation guaranteed by the actor model.
```

Comment:
```suggestion
This proposal introduces a basic mechanism for customizing actor  executors. By providing an instance of an executor, actors can influence "where" they will be executing any task they are running, while upholding the mutual exclusion and actor isolation guaranteed by the actor model.
```

Reply:
Almost, this is mean to be mutual exclusion; as in "mutually exclusive"

Thanks for spotting the typo!